mmdet/datasets/utils.py

```diff
@@ -67,3 +70,22 @@ def show_ann(coco, img, ann_info):
     plt.axis('off')
     coco.showAnns(ann_info)
     plt.show()
+
+
+def get_dataset(data_cfg):
+    if isinstance(data_cfg['ann_file'], list) or \
+            isinstance(data_cfg['ann_file'], tuple):
+        ann_files = data_cfg['ann_file']
+        dsets = []
+        for ann_file in ann_files:
+            data_info = copy.deepcopy(data_cfg)
```

Comment:
Besides `ann_file`, there are other fields that also needs to be split such as `proposal_file`.